Memeolist/Memeolist/AppDelegate.swift

```diff
@@ -15,17 +15,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // create the authentication config
         let authenticationConfig = AuthenticationConfig(redirectURL: "memeolist://callback")
-        try! AgsAuth.instance.configure(authConfig: authenticationConfig)
-        if let transport = AgsSync.instance.transport {
-            transport.headerProvider = AgsAuth.instance.getAuthHeaderProvider()
-        }
-        guard let _ = try! AgsAuth.instance.currentUser() else {
-            let rootViewController = LoginViewController()
-            window?.rootViewController = UINavigationController(rootViewController: rootViewController)
-            window?.makeKeyAndVisible()
-            return true
+        do {
+            try AgsAuth.instance.configure(authConfig: authenticationConfig)
+            if let transport = AgsSync.instance.transport {
+                transport.headerProvider = AgsAuth.instance.getAuthHeaderProvider()
+            }
+            guard let _ = try! AgsAuth.instance.currentUser() else {
+                let rootViewController = LoginViewController()
+                window?.rootViewController = UINavigationController(rootViewController: rootViewController)
+                window?.makeKeyAndVisible()
+                return true
+            }
+        } catch AgsAuth.Errors.noServiceConfigurationFound {
+            print("No Auth configuration found.")
+        } catch {
+            print("Some other error.")
```

Comment:
When there is error with token or something else we may just show login screen.

Reply:
ok